lib/fog-local.rb

```diff
@@ -1 +0,0 @@
-require 'fog/local'
```

Comment:
Does this cause performance issues? I don't think it has been a convention with other fog provider gems, but it's nice to be able to automatically require a gem via Gemfile/bundler.